app/src/main/java/org/oppia/android/app/notice/OsDeprecationNoticeDialogFragmentPresenter.kt

```diff
@@ -30,7 +30,7 @@ class OsDeprecationNoticeDialogFragmentPresenter @Inject constructor(
         )
       )
       .setNegativeButton(R.string.os_deprecation_dialog_dismiss_button_text) { _, _ ->
-        deprecationNoticeExitAppListener.onActionButtonClicked(
+        deprecationNoticeActionListener.onActionButtonClicked(
           DeprecationNoticeActionType.DISMISS
```

Comment:
The OS being deprecated means action close I think? the app shouldn't be opened, I can see from the demo video that the profile chooser screen is launched.

Reply:
From the design doc and the product requirements, the OS deprecation only prevents you from updating the app. It shouldn't stop you from using the app that you already have.

Reply:
That actually makes sense. Thanks!